### DISCUSSION TITLE
RTranslator 2.0.4

### DISCUSSION THREAD

**Discussion by niedev** [niedev/RTranslator#55] [General]
The app is now localized in Traditional Chinese (thanks to @abc0922001) and Simplified Chinese (thanks to @starlightjy)

The app icon is now compatible with Material You themes (thanks to @GitGitro).

Muting the microphone now no longer cancels the results but only stops the listening (so it can be used to force stop listening in noisy environments).

<hr /><em>This discussion was created from the release <a href='https://github.com/niedev/RTranslator/releases/tag/2.0.4'>RTranslator 2.0.4</a>.</em>

**Comment by fishergz** [comment]
Significantly faster than 2.0.2 in my Mi10
Thanks.

**Comment by eric52zhang** [comment]
1.2g的模型无法完全下载

**Comment by niedev** [comment]
@eric52zhang China has [some problems](https://www.reddit.com/r/China/comments/v8fv0p/why_is_github_so_slow_in_china_recently/) with the download, to solve this you could try to do the [sideloading](https://github.com/niedev/RTranslator/blob/v2.00/Sideloading.md).

**Reply by HZQ0556** [reply]
I followed the dedicated guide, but there's no "nie.translator.rtranslator" folder in my phone. I wonder why o(╥﹏╥)o

**Comment by jiangzise2019** [comment]
I have downloaded the 1.2GB model package, but when I use RTranslator, it still prompts' Error occurred during TTS initialization ', which means I can only use no TTS mode.And this app is very unstable on my phone, with frequent crash when used lightly.My phone model is MOTO X40, Android 14

**Reply by niedev** [reply]
That's strange, your phone have 8GB or 12GB of RAM? (For the TTS problem I will update the readme today with an explanation to solve it)

**Reply by jiangzise2019** [reply]
12GB RAM

**Reply by niedev** [reply]
Could you show me information about your phone's RAM usage with RTranslator in the background? (just open it and then press the home button) To find the setting that shows RAM usage you should go into your phone settings and search for "memory".

**Reply by jiangzise2019** [reply]
last 3 hour
Average memory use :142MB;
Frequency:Rarely running(7%);
Maximun usage:2.1GB;
RTranslator:140MB/:Rarely running(7%);
Z-RAM:1.5MB/:Rarely running(7%).

**Reply by niedev** [reply]
Can you give me a screenshot like this?

![image](https://github.com/user-attachments/assets/83350b75-3157-4285-b8aa-ae1df9ce11b8)

And another like this (for this screen you should click on the three dots on the top right and click on "Sort by max use"):

![image](https://github.com/user-attachments/assets/7f2bd5ad-8e4b-4c5c-b382-512247cf4ce5)

Because I need to know how much free RAM you have and if other apps consumes a lot of RAM to understand if the crashes are caused by too little free RAM or for some bug

**Comment by jiangzise2019** [comment]
okk，
![微信图片_20240726023642](https://github.com/user-attachments/assets/a7597b35-dd02-40ce-85c2-db8b2ee0cb17)
![微信图片_20240726023652](https://github.com/user-attachments/assets/557e0384-7caf-4dd2-8446-49dd60445dc8)
![微信图片_20240726023708](https://github.com/user-attachments/assets/89632836-2638-4763-a4f5-1237d18aaa9a)

**Reply by niedev** [reply]
On the "memory used by apps" screen you show the average memory, but I need the max memory used, if you click on the tree dots in the top right, is there a "Sort by max use" option? If yes click on it and take a screenshot

**Reply by jiangzise2019** [reply]
Memory data after three minutes of using RTranslator

![微信图片_20240726105726](https://github.com/user-attachments/assets/f874a72b-fc0a-4ff7-a989-be5ef8f0bd92)
![微信图片_20240726105731](https://github.com/user-attachments/assets/f032e9ed-c1f7-4347-9bb6-32a3ef9faee8)

**Reply by niedev** [reply]
It seems to me that your phone has too many apps in RAM (10 GB of RAM used out of 12 is not normal, my phone even with RTranslator open uses 6 out of 12), normally in these cases when you start a heavy app like RTranslator the operating system should free up the memory by closing the other apps, but it seems that for some reason this does not happen (probably if you start a heavy game on your phone you will have the same problems as with my app), I will find out what I can do on my side to better manage RAM allocation, but I warn you that there is most likely not much I can do.  For now the only fix I can think of is to clean all recent apps before starting RTranslator, or, even better, limit the background use of some of the heavier apps (like TikTok) by deactivating notifications or activating limitations for energy saving on the latter (I don't know where these settings are in your phone but they are usually in a section dedicated to battery in the phone settings or in settings -> applications -> TikTok (or any app you want to limit))

**Comment by Z1298434831** [comment]
模型没法下载

**Reply by niedev** [reply]
As I say in the readme the download from china has some problem, try [sideloading](https://github.com/niedev/RTranslator/blob/v2.00/Sideloading.md)

**Comment by xingyun214XX** [comment]
这个对话模式是需要先和另一部手机建立链接后才能使用的吗？
